libpod/networking_linux.go

```diff
@@ -943,21 +945,39 @@ func getContainerNetIO(ctr *Container) (*netlink.LinkStatistics, error) {
 		return nil, nil
 	}
 
-	// FIXME get the interface from the container netstatus
-	dev := "eth0"
 	netMode := ctr.config.NetMode
+	netStatus := ctr.getNetworkStatus()
 	if otherCtr != nil {
 		netMode = otherCtr.config.NetMode
+		netStatus = otherCtr.getNetworkStatus()
 	}
 	if netMode.IsSlirp4netns() {
-		dev = "tap0"
+		// create a fake status with correct interface name for the logic below
+		netStatus = map[string]types.StatusBlock{
+			"slirp4netns": {
+				Interfaces: map[string]types.NetInterface{"tap0": {}},
+			},
+		}
 	}
 	err := ns.WithNetNSPath(netNSPath, func(_ ns.NetNS) error {
-		link, err := netlink.LinkByName(dev)
-		if err != nil {
-			return err
+		for _, status := range netStatus {
+			for dev := range status.Interfaces {
+				link, err := netlink.LinkByName(dev)
+				if err != nil {
+					return err
+				}
+				if netStats == nil {
+					netStats = link.Attrs().Statistics
+					continue
+				}
+				// Currently only Tx/RxBytes are used.
+				// In the future we should return all stats per interface so that
+				// api users have a better options.
```

Comment:
nit of a nit.  Ignore unless you need to update something else.
```suggestion
				// api users have  better options.
```